assignments/array-methods.js

```diff
@@ -55,27 +55,49 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 {"id":50,"first_name":"Shell","last_name":"Baine","email":"sbaine1d@intel.com","shirt_size":"M","company_name":"Gabtype","donation":171}];
 
 // ==== Challenge 1: Use .forEach() ====
-// The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
+// The event director needs both the first and last names of each runner for their running bibs.
+// Combine both the first and last names into a new array called fullName.
+
+console.log('CHALLENGE 1 👇');
+
 let fullName = [];
+
+runners.forEach(function(runner) {
+    fullName.push(runner.first_name + " " + runner.last_name);
+  });
+
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
-// The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
-let allCaps = [];
+// The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER.
+// Convert each first name into all caps and log the result
+
+console.log('CHALLENGE 2 👇');
+
+let allCaps = runners.map(runner => (runner.first_name.toUpperCase()));
```

Comment:
Good work here. You declared a new identifier and assigned it to a function in one line and using the ES6 arrow function.